cluster/images/etcd/Makefile

```diff
@@ -30,7 +30,7 @@ TAGS?=2.2.1 2.3.7 3.0.17
 REGISTRY_TAG?=3.0.17
 ARCH?=amd64
 REGISTRY?=gcr.io/google_containers
-GOLANG_VERSION?=1.7.5
+GOLANG_VERSION?=1.8.1
```

Comment:
apparently, v3.2 will target v1.8.1: https://github.com/coreos/etcd/issues/6174

This was kind of an automated bump; happy to still use v1.7 while we're on v3.0.x (and v3.1.x)